magley/klau-drive
# 2.1 serverless

Closes #10 

How to test:

You have to manually type in the commands for creating each lambda and setting its timeout.
In `src/lambdas/lambda_*.py`, there are block-comments containing the specific commands for creating, updating timeout, updating code and invoking the lambda functions from the command line. Call the first two commands for each lambda function.
For updating, there are 2 python scripts: `zip.py` and `deploy.py`. Check `src/lambdas/README.md`. 

After that, just run the app, it should work from that point(?).

I'll make utility functions for creating lambdas some other time.
Also, there is some code redundancy, especially with the `boto3` clients and `create_*_if_not_exist`.
